audio/decode/ad_lavc.c

```diff
@@ -156,7 +156,7 @@ static bool init(struct mp_filter *da, struct mp_codec_params *codec,
     return true;
 }
 
-static void destroy(struct mp_filter *da)
+static void ad_lavc_destroy(struct mp_filter *da)
```

Comment:
I don't see why prefixing static local functions is required?

Reply:
> but only process is shown in the stack traces and profilers.

Generally you have associated compilation uinit and even line number in this case to differentiate. But of course it doesn't harm to name the functions directly.